packages/api/src/schema/schema.yaml

```diff
@@ -1423,6 +1409,22 @@ components:
           example: filename.mp4
         playbackPolicy:
           $ref: "#/components/schemas/playback-policy"
+        storage:
+          additionalProperties: false
+          properties:
+            ipfs:
+              oneOf:
+                - type: object
+                  additionalProperties: false
+                  properties:
+                    spec:
+                      oneOf:
+                        - type: "null"
```

Comment:
Just out of curiosity, I saw it was already like that, but Is this how the null type is defined here? Is it required to be a string in quotation marks?

Reply:
Hmm, actually for YAML it's just a string, but I think `null` is a special keyword since it also exists in JSON. If we used `null` here, this object would have a `{"type": null}` field (no type on JSON) instead of a `{"type": "null"}` (`null` type on JSON schema). [ref](https://stackoverflow.com/a/21121267/1560458)